content/customize-dock.md

```diff
@@ -23,17 +24,17 @@ redirect_from:
   - /articles/move-launcher
 ---
 
-Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen. You can adjust this preference, move the dock to the bottom of the screen, and more in <u>Settings</u>.
+Pop!\_OS and Ubuntu both contain an application dock on the screen. You can adjust this preference, such as move the dock to a different side the screen, and more in <u>Settings</u>.
 
 ## Default Dock on Pop!\_OS
 
-Pop!\_OS includes a lightweight version of Dash to Dock, which is why the dock already displays on the bottom of the screen by default.
+Pop!\_OS includes a lightweight version of Dash to Dock like in the screenshot below:
```

Comment:
"the screenshot below" wasn't mentioned before. Why was that added? 

Also I don't think either COSMIC Dock or Ubuntu's dock are any lighter than Dash to Dock, so I'm not so sure "lightweight" has any basis in this note or the one below. But I've been overlooking that on my past reviews, so if you want to leave it, I won't complain too much. I'm just uncertain about it.